header.go

```diff
@@ -513,6 +514,85 @@ func fixUnquotedSpecials(s string) string {
 	return clean.String()
 }
 
+func fixUnescapedQuotes(s string) string {
+	split := strings.SplitAfter(s, ";")
+	fresh := &strings.Builder{}
+	for i := 0; i < len(split); i++ {
+		// find the "="
+		idx := strings.IndexByte(split[i], '=')
```

Comment:
```suggestion
		eq := strings.IndexByte(split[i], '=')
```

I don't see `idx` used in Go; for example, it's not used once in the entire strings pkg.  I do think that the Go authors sometimes overuse the single character variables which can be confusing in longer functions.

Reply:
Huh, I must have carried that over from my C# or Java days, lol. Good point. I believe that shorter var names would make the intermediate AST smaller during compile time, but in practice I think it makes it easier to see the business logic abstracted away from the preconceptions that wordy var names can imply... I don't practice this enough, so I digress.